src/main/java/sirius/biz/protocol/DelegateJournalData.java

```diff
@@ -118,6 +122,21 @@ public void setSilent(boolean silent) {
         this.silent = silent;
     }
 
+    /**
+     * Adds an entry to the journal of the given entity.
+     *
+     * @param targetType the type of the entity under which the journal entry will be created
+     * @param targetId   the id of the entity under which the journal entry will be created
+     * @param aClass     the class of the actual entity which was changed
+     * @param changes    the entry to add to the journal, automatically appended by the class's label
+     */
+    public static void createJournalEntry(String targetType, String targetId, Class<?> aClass, String changes) {
```

Comment:
Class<?> aClass
```suggestion
    public static void createJournalEntry(String targetType, String targetId, Class<?> contentIdentifierClass, String contentIdentifierId, String changes) {
```

Reply:
For the direct deletion via (like via Mongo), we have no ID to provide if we don't make a lookup before...